flagging_site/templates/api/index.html

```diff
@@ -26,4 +27,52 @@ <h2>Available Endpoints</h2>
         <li><a href="{{ url_for('api.reachesapi') }}" target="_blank">Flagging Model Outputs</a></li>
         <li><a href="{{ url_for('api.boathousesapi') }}" target="_blank">Boathouses</a></li>
     </ul>
+    <h2>Sample code to call API</h2>
+    <ul>
+        <li>Calling Flagging Model Outputs</li>
+        <ul>
+            <li>
+                using Python
+                <pre class="prettyprint">
+                    import pandas as pd   
+                    url = "http://127.0.0.1:5000/api/v1/model"
+                    model_output = pd.read_json(url)
+                    print(model_output.head())
```

Comment:
In general HTML ignores multiple whitespaces without `&nbsp;`, but here because of how the javascript tool works, it reads all the whitespace.

<img width="902" alt="image" src="https://user-images.githubusercontent.com/31971762/95001732-dca78280-059a-11eb-896f-eb93015a8986.png">

The simplest solution to making it not have all that whitespace is to just delete it. It makes the HTML file itself less pretty to delete the whitespace, but it makes the website look better, and the latter is more important.